pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.google.sps</groupId>
+    <artifactId>step240-2020</artifactId>
+    <version>1.0</version>
+    <packaging>war</packaging>
+
+    <dependencies>
```

Comment:
Consider adding JUnit or another testing framework as a dependency

Reply:
I think, it would be better if we add necessary dependencies only when we actually need them